wdae/wdae/users_api/tests/test_users_authentication.py

```diff
@@ -181,12 +182,13 @@ def test_failed_auth_lockouts(db, user, client, tokens):
         regex = r"locked out for (\d+) hours and (\d+) minutes"
         match = re.search(regex, response.content.decode())
         response_hours, response_minutes = map(int, match.groups())
-        total_minutes = pow(2, i)
-        hours = int(total_minutes / 60)
-        minutes = total_minutes % 60
+        response_td = timedelta(
+            hours=response_hours, minutes=response_minutes
+        )
+        expected_td = timedelta(minutes=pow(2, i))
 
-        assert response_hours == hours
-        assert response_minutes == minutes
+        # Give a tolerance of one minute to prevent test from becoming flaky
```

Comment:
Was the test flaky enough to demand approximation up to a whole minute?

Reply:
I figure minute precision is good enough for testing - didn't want to experiment with 2 or 5 second tolerances

Reply:
But if we see a difference that is almost a minute wouldn't that mean that there is an issue with the code? I think 5 seconds should be more than enough

Reply:
Alright - I'll change the tolerances to 5 seconds.